README.md

```diff
@@ -23,16 +23,15 @@ instance:
   VERSION=0.1.0
   NAMESPACE=default
   GITEA_ENDPOINT="http://gitea-http.${NAMESPACE}:3000/"
-   GITEA_ADMIN_USERNAME=#Define a username for the admin
-   GITEA_ADMIN_PASSWORD=#Define a password for the admin
+  GITEA_ADMIN_USERNAME=#Define a username for the admin
+  GITEA_ADMIN_PASSWORD=#Define a password for the admin
```

Comment:
Please keep at lest the password intended as a leading whitespace will not save the command in the history of bash

Reply:
Oh, I didn't know that. Added the whitespaces again.